README.md

```diff
@@ -1,29 +1,145 @@
-# kubevirt-ansible
+# KubeVirt Ansible
 
 This repository provides collection of playbooks to
-* install KubeVirt on existing OpenShift cluster
-* deploy K8S on given machines and install KubeVirt
-* deploy OpenShift on given machines and install KubeVirt
+- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
+- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
+- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
+- [ ] Deploy Kubernetes cluster with KubeVirt with Lago
+- [x] [Deploy OpenShift cluster with KubeVirt with Lago](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
```

Comment:
I don't think that we should create a dedicated playbook for this task.
Please see my comment about `control.yaml`.

Reply:
@gbenhaim  I can merge it together and rephrase it as I did for `control.yml` description.

Reply:
+1